src/libaktualizr/package_manager/dockerappmanager.cc

```diff
@@ -158,8 +158,8 @@ data::InstallationResult DockerAppManager::install(const Uptane::Target &target)
     LOG_INFO << "Pruning old docker artifacts";
```

Comment:
Mind also updating here to log that it is now pruning unused docker images?

Reply:
Ah yes.